src/iceberg/v1_metadata.cc

```diff
@@ -19,6 +19,9 @@
 
 #include "iceberg/v1_metadata.h"
 
+#include <nlohmann/json.hpp>
+
```

Comment:
```suggestion
```

I don't think we need this.

Reply:
json_internal only include #include <nlohmann/json_fwd.hpp>, so this cc must include this header since `ToJson` returns a  nlohmann::json

Reply:
OK, let's include it in the cc file.